bandit/core/config.py

```diff
@@ -23,6 +24,7 @@
 from bandit.core import utils
 
 
+BASE_CONFIG = 'bandit.yml'
```

Comment:
This is in disagreement with README file, where name `.bandit.yml` is used.

Reply:
OK, there are two configuration files.  One, .bandit is to put command line options.  The other, the bandit.yml via -c is for more extensive options.

However this patch is making it even more confusing.

Reply:
https://github.com/PyCQA/bandit/pull/458#issuecomment-468849496